### PR TITLE
Make EditorServicesTest2 a UnitTestNext project.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
This way we can make the tests in it test against OOP.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @Pilchie

File: src/Workspaces/Remote/ServiceHub/project.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
     "dependencies": {
-        "StreamJsonRpc": "0.12.32-alpha-g90be50f449"
+        "StreamJsonRpc": "1.0.2-rc"
```
Comment (Pilchie) [inline src/Workspaces/Remote/ServiceHub/project.json]: Also @heejaechang is this okay to do?

File: src/EditorFeatures/Test2/EditorServicesTest2.Next.vbproj [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <OptionStrict>Off</OptionStrict>
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <TargetFrameworkProfile />
-    <RoslynProjectType>UnitTest</RoslynProjectType>
```
Comment (Pilchie) [inline src/EditorFeatures/Test2/EditorServicesTest2.Next.vbproj]: You will need a Roslyn.sln update for this.  Right now, building Roslyn.sln isn't going to build this...

Not having it is also the reason for your ETA failure.

File: src/Workspaces/Remote/ServiceHub/project.json (hunk 1 above)
Comment (heejaechang) [inline src/Workspaces/Remote/ServiceHub/project.json]: our base drop is now preview 5? if so, it should be fine.

Comment (Pilchie) [inline src/Workspaces/Remote/ServiceHub/project.json]: Yes, @jaredpar and @tmeschter are working on requiring preview 5 in our test environments.

**Comment by tmeschter** [comment]
Our build systems have Preview3, but you've taken a dependency on Preview5.

**Comment by CyrusNajmabadi** [comment]
Do you know when we might be moving to a later build of VS?

**Comment by tmeschter** [comment]
As soon as I can make it happen. Unfortunately it is complicated.

**Comment by Pilchie** [comment]
Does this supercede #15035?

**Pull request closed without merging**